CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 2.2.0-beta3
 - New property `DefaultCounters` in `PerformanceCollectorModule` to control the list of standard counters that will be collected
 - Default performance counters will be reported as metrics
+- By default `DependencyTrackingTelemetryModule` will not detect certain http dependencies as Azure Storage calls. You need to register a telemetry initializer `HttpDependenciesParsingTelemetryInitializer` to enable this functionality. 
```

Comment:
This is confusing... You are adding it by default in config. I code-based instrumentation is used than you do not get it by default.